### ISSUE TITLE
Update documentation to reflect use in new Lightning Professional Edition

### ISSUE THREAD

**Issue by m-a-t-m** [afawcett/declarative-lookup-rollup-summaries#341]
It is now possible to install the managed package in a Lightning Professional Edition and configure entries to trigger Process Builder to action the declarative roll up summary.

**Comment by afawcett** [comment]
Yes! Yes! Yes! 👍

**Comment by JimBTek** [comment]
added to wiki page but not readme

**Comment by afawcett** [comment]
Thanks @JimBTek 👍

**Comment by mattischvili** [comment]
Great thing that your amazing tool now is available also for Professional Editions.

I can't figure how to do it, however.
1. First of all, I am not able to deploy the code for the child trigger. But that might be the thing that is not possible to do in PE?
2. I have selected Calculation mode = Process Builder. But do I need to do anything else? Create some kind of process in Process Builder?

If you just point me in the right direction I would, after succeeding, be happy to update the Wiki to help other lost souls.

**Comment by afawcett** [comment]
RE: 1, you cannot do this in PE and you don't need to if your using Process Builder.

RE: 2, yes you need to create Process Builder as defined in this [blog here](https://andyinthecloud.com/2015/02/16/declarative-lookup-rollup-summaries-tool-dlrs-spring15-release/), make sure the rollup is marked as Active.

**Comment by afawcett** [comment]
README updated.

**Comment by faceroy** [comment]
Hi Andy,

Followed the blog post (in relation to utilising in a PE org) but unable to see the call Apex option for actions.

Any ideas?

Thanks in advance

**Comment by afawcett** [comment]
That is odd, can you send a screenshot?

**Comment by faceroy** [comment]
Sure, here you go... Thanks for your assistance.

<img width="1238" alt="screen shot 2017-03-14 at 21 45 48" src="https://cloud.githubusercontent.com/assets/14908254/23923751/d27d607c-08ff-11e7-8e53-5cd2c75ddab1.png">
<img width="1178" alt="screen shot 2017-03-14 at 21 46 57" src="https://cloud.githubusercontent.com/assets/14908254/23923762/e1a72024-08ff-11e7-84da-f4e6e0bdb870.png">

**Comment by afawcett** [comment]
Thats really odd, your missing the Apex Action action type? I have never seen that before. . I wonder if there is some restriction in PE orgs? If so, i wonder how i have ever got reports of DLRS working in PE tbh? The documentation seems to suggest it is [fine in PE though](https://help.salesforce.com/articleView?id=process_action_apex.htm&language=en_US&type=0).... 

This is what i see, though this is in a DE org...

![image](https://cloud.githubusercontent.com/assets/1167760/24082604/907e8636-0c85-11e7-90fb-32e7523b001c.png)

I did some searching, not 100% sure, but wondering if its this [known issue](https://success.salesforce.com/issues_view?id=a1p300000008ZFDAA2&title=process-builder-apex-action-disappears-when-apex-class-is-modified-to-include-reference-to-some-standard-fields)??? The above documentation link also notes this... "If you define an Apex action in your process and then modify the Apex class by adding a standard field reference (for example, User.Phone), the Apex action is no longer visible in the process and must be added again."

Clearly though you are not writing an invocable method or for that matter changing it.... So it does not 100% add up to me yet. I also see from your screenshot your missing a few other options. I think its worth asking Salesforce support why your missing this feature. Does it occur when you create a brand new process as well btw?

**Comment by afawcett** [comment]
Check out the continued discussion on https://github.com/afawcett/declarative-lookup-rollup-summaries/issues/478...

**Comment by matti-sch** [comment]
@m-a-t-m, could you explain how you got this to work PE (even if in Lightning)? Would be really appreciated.

**Comment by m-a-t-m** [comment]
12 months ago, in May 2016 I did configure a PE org using DLRS and Process Builder with the Apex invokable action.  Then it stopped working.
I tracked the cause in my org to the following known issue (WHICH IS STILL OPEN 12 MONTHS LATER!?)  
https://success.salesforce.com/issues_view?id=a1p300000008ZFDAA2&title=process-builder-apex-action-disappears-when-apex-class-is-modified-to-include-reference-to-some-standard-fields
Could this be the issue for you too @matti-sch ?

**Comment by matti-sch** [comment]
It sure sounds like it. Thank you for sharing.

Den 25 apr. 2017 8:28 em skrev "m-a-t-m" <notifications@github.com>:

**Comment by m-a-t-m** [comment]
Is it possible to work around the known issue @afawcett by removing references to the standard fields inside DLRS classes?  I am guessing probably not, you wouldn't have them in there if they were not needed.

**Comment by chadwix** [comment]
For the benefit of posterity, we're a SF Lightning Professional Edition organization and were seeking to sum a field from child Accounts to each Parent Account (which are territories, the way we use SF). 

**To get this to work -- which it does wonderfully** -- we followed the installation instructions [here](https://www.dandonin.com/2017/03/16/how-to-dlrs/) (although the version that article links to is outdated), then did the following:

- Left "Active" unselected when we first created the Rollup
- Set Calculation Mode to "Process Builder"
- Once we'd saved the Rollup, we went back and selected "Active", then resaved
- We tried clicking "Manage Child Trigger" (accessed by clicking the top-right menu arrow next to the Action Buttons). When we did this, SF threw an error.
- On a whim, we clicked the "Calculate" action just above the "Manage Child Trigger" action, and the Rollup worked.

Not that, although we built a Process in Process Builder, we tested out what would happen if we deactivated it, and the Rollup continued to work and recalculate as we expected when we ran some tests.

Hope this helps anyone in the future.
